lib/efax/outbound.rb

```diff
@@ -23,6 +23,8 @@ module EFax
   # Prefered content type
   HEADERS  = {'Content-Type' => 'text/xml'}
 
+  OUTBOUND_RESOLUTION = ENV['EFAX_OUTBOUND_RESOLUTION'] || "FINE"
```

Comment:
Perhaps to keep this consistant with the other efax configurations we move this into requests as a configuration option rather then reading from ENV (since this gem doesn't use ENV any where else).

So we we sould do something like the `def self.user` blocks, or perhaps we add a configuration object for Transmission control that we can pass into `self.xml`  for resolution, priority etc.

Reply:
That makes sense. I'll do that.